### PR TITLE
docs

### PR DESCRIPTION
**Pull request by LiliC**
Add some note about where to find release images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ Start the Habitat operator by running:
 
 ### Running inside a Kubernetes cluster
 
+### Building image from source
```
Comment (krnowak) [inline README.md]: This makes the "Running inside a Kubernetes cluster" section empty. Maybe you wanted to use four hashes for the "Building image from source" section?

File: README.md [hunk 2]
```diff
@@ -40,6 +42,10 @@ This will produce a `kinvolk/habitat-operator` image, which can then be deployed
 
 The name of the generated docker image can be changed with an `IMAGE` variable, for example `make image IMAGE=mycorp/my-habitat-operator`. If the `habitat-operator` name is fine, then a `REPO` variable can be used like `make image REPO=mycorp` to generate the `mycorp/habitat-operator` image. Use the `TAG` variable to change the tag to something else (the default value is taken from `git describe --tags --always`) and a `HUB` variable to avoid using the default docker hub.
 
+### Using release image
```
Comment (krnowak) [inline README.md]: Four hashes?

**Comment by LiliC** [comment]
@krnowak good point, wdyt if we change the `Topology` scheme to string, as it is a string in the end?

**Comment by krnowak** [comment]
I'd say that Topology is more like an enumeration, but you already document the possible values. So yeah, string sounds fine. A problem that I see is that we allow either "standalone" or "leader" topologies, but there is also "none" in Habitat. I need to check what is the difference between the "standalone" topology and "none".

**Comment by LiliC** [comment]
@krnowak IIRC topology in Habitat itself always defaults to standalone.

**Comment by LiliC** [comment]
@krnowak changed to `string`. PTAL.

**Comment by tashimi** [comment]
Topology defaults to none, @fnichol can weigh in too, the difference between standalone and none is that none will never update itself

**Comment by LiliC** [comment]
@tashimi Interesting, as when I was reading about topologies, I came across [this](https://www.habitat.sh/docs/glossary/#glossary-services) in the docs at some point so I relied on that info.
> Standalone
>This is the default topology, useful for services inside a group that are completely independent from one another. 

If that's the case we will make the adjustment in the operator. Thanks for this!

**Comment by tashimi** [comment]
I meant to update those docs when I was going through them a couple weeks ago but didn’t have time - still on the to do list. 😬

**Comment by LiliC** [comment]
@tashimi No worries, I created an issue for it and will get to it asap. Thanks again!

**Comment by krnowak** [comment]
I suppose that in our case there's no difference between none and standalone topology since we do not use any update strategy.

Anyway, this PR LFAD. (looks fine and dandy)

**Pull request merged**